paperless.go

```
@@ -752,6 +753,120 @@ func (client *PaperlessClient) DownloadDocumentAsImages(ctx context.Context, doc
+			// File exists
+			pdfPaths = append(pdfPaths, pdfPath)
+		}
+	}
+
+	// If all PDFs exist, return them
+	if len(pdfPaths) == pagesToProcess {
+		return pdfPaths, pdfData, totalPages, nil
+	}
+
+	// Clear existing PDFs to ensure consistency
+	pdfPaths = []string{}
```

golangci-lint (1.64.8)

ineffectual assignment to pdfPaths

(ineffassign)